src/Catalog/sparql/ConstructRegistrationPageContentGraph.rq

```diff
@@ -11,7 +11,7 @@ CONSTRUCT
     ?catalogEntry a nuget:PackageDetails ;
                   nuget:id ?id ;
                   nuget:version ?version ;
-	              nuget:published ?published ;
+				  nuget:published ?published ;
```

Comment:
This still has tabs for indentation. Let's just align the indention with `nuget:version ?version ;` using spaces only

Reply:
Done